pkg/ruler/rules/local/local.go

```diff
@@ -143,7 +145,7 @@ func (l *Client) listAllRulesGroupsForUser(ctx context.Context, userID string) (
 func (l *Client) listAllRulesGroupsForUserAndNamespace(ctx context.Context, userID string, namespace string) (rules.RuleGroupList, error) {
 	filename := filepath.Join(l.cfg.Directory, userID, namespace)
 
-	rulegroups, allErrors := rulefmt.ParseFile(filename)
+	rulegroups, allErrors := l.loader.Load(filename)
```

Comment:
Note to myself: in Cortex we always use `FileLoader` whose `Load()` function calls `rulefmt.ParseFile()`, so this is a no-op.